### ISSUE TITLE
Adding JOSS to Web of Science / Clarivate

### ISSUE THREAD

**Issue by chadagreene** [openjournals/joss#1283]
[Creating a new issue from [this discussion on PubMed indexing](https://github.com/openjournals/joss/issues/153).] 

JOSS is not currently included in the [Web of Science / Clarivate master list of journals](https://mjl.clarivate.com/search-results). This is a problem because my employer's annual salary review process only considers articles that are indexed by Web of Science. Can we get JOSS included in the WoS master list?

**Comment by arfon** [comment]
As noted here, we've (re)submitted: https://github.com/openjournals/joss/issues/153#issuecomment-1566213527

**Comment by chadagreene** [comment]
Thanks @arfon. The resubmission was back in May, which is why we're inquiring about updates. A video on the [WoS Publisher Portal](https://clarivate.com/products/scientific-and-academic-research/research-publishing-solutions/publisher-portal/) says there's an Evaluation Tracker that should provide some insights about the current status of the submission. Does it tell us anything?

**Comment by arfon** [comment]
Not much sorry:

<img width="1203" alt="Screenshot 2023-10-30 at 08 24 31" src="https://github.com/openjournals/joss/assets/4483/1aee96ed-8ace-433c-99b1-a95458c17c0a">

**Comment by chadagreene** [comment]
I just reached out to Clarivate to check on the status and got this response: 

> Dear Chad, 
> 
> Thank you for contacting Clarivate. 
> 
> I understand that you want to know the evaluation status of the journal “Journal of Open Source Software”, ISSN: 2475-9066 in Web of Science Publisher Portal.
> 
> Upon checking, I noticed that the journal has been recently rejected. Rejection emails has been sent out from the email of  [alerts-noreply@clarivate.com](mailto:alerts-noreply@clarivate.com) and has be sent to who was listed as the Editorial Contact ([admin@theoj.org](mailto:admin@theoj.org)). The Editorial contact may also track the status of the journal on the Publisher Portal dashboard.
> 
>  I hope this helps.
> 
> Please do not hesitate to contact us if you have any further questions.
> 
> I will be happy to assist you.
> 
> Regards

**Comment by sneakers-the-rat** [comment]
the journal review criteria: https://clarivate.com/products/scientific-and-academic-research/research-discovery-and-workflow-solutions/webofscience-platform/web-of-science-core-collection/editorial-selection-process/editorial-selection-process/

Initial Triage:
- [x] Have an ISSN - [yep](https://portal.issn.org/resource/ISSN/2475-9066)
- [x] Have a title
- [x] Have a publisher
- [x] Have a URL   
- [x] Have access to content
- [x] Presence of peer review policy
- [x] Have contact details

Editorial Triage:
- [x] Primarily scholarly content
- [x] Article titles and abstract in english
- [x] Bibliographic information in roman script
- [x] Clarity of language in published work
- [x] Have a publication schedule that is adhered to (for JOSS it's just continuous), and have published "enough" work
- [x] Website must be accurate and easy to navigate and link to publisher website and vice versa
- [x] Presence of ethics statements - [yep](https://joss.readthedocs.io/en/latest/policies.html)
- [x] Editorial board members must be identifiable and available for contact
- [x] Authors need to be identified with affiliations - idk if we enforce this but it's certainly there.

Editorial evaluation (quality)
- [x] Editor and editorial board geographical and other diversity, consistent with volume and breadth of output
- [x] Published content should do the things the policy says it should do
- [x] "Published content must reflect adequate peer review and editorial oversight - signs of deficient peer review include articles that demonstrate lack of scholarly rigor or validity, presence of articles outside the scope of the journal" 
- [x] Journal must have scope, and papers must be within that scope
- [x] If supported by a grant, say so
- [x] Adherence to community standards - "editorial policies are consistent with recognized best practices such as COPE Core Practices" - idk joss basically invented the best practices for open peer review of software so...
- [x] Author diversity, geographic, institution, etc.
- [x] Appropriate citations to surrounding literature

Editorial evaluation (impact)
- [x] Papers have to be cited a lot - [Dimensions](https://app.dimensions.ai/discover/publication?search_mode=content&or_facet_source_title=jour.1156086) says that there are 56K citations with a mean of 24.22, which rocks, including such "everyone knows them" works like the tidyverse, umap, seaborn, and the like.  
- [x] Most authors should have a publication record also in WoS - I mean i can only assume, they certainly are showing up in dimensions as being recognized
- [x] Most editorial board members should have a publication history in WoS - again like.. yes?
- [x] "Content significance" - content should be important, unique or "enrich the breadth of WoS coverage"

So i would love to know the reason they rejected us, because we easily pass every check

**Comment by rossmounce** [comment]
rather than continue to beg the black box for a different arbitrary decision, or another roll of the dice... Perhaps the JOSS community might want to turn the attention to working on eliminating "Does the journal have a Clarivate assigned Journal Impact Factor™?" from hiring, promotion, tenure, and salary review processes, wherever those thought-bunkers may be found?

At the end of the day Clarivate is fully entitled to choose an arbitrary list of journals to annoint with a proprietary and statistically illiterate number (the Journal Impact Factor™) and also which journals they care NOT to give a proprietary and statistically illiterate number to.  The strategy to approach this problem needs to be different. 

Time to help the organizational users of these statistically illiterate numbers and arbitrary inclusion/non-inclusion journal lists to see sense?

**Comment by sneakers-the-rat** [comment]
To be clear the last thing i was suggesting was that they were a fair player that followed their rules, just pointing out how obviously that is the case. I think in this case OP is saying they have a direct need in order to be able to publish their work in JOSS, but i also think we can 'do both' - continue trying to replace the world of proprietary metrics and also hassle and mock them about their editorial practices

**Comment by sneakers-the-rat** [comment]
Can we get clarification/confirmation from anyone who would know whether we received notice or any justification from clarivate re: refusal to index?

**Comment by MikeTaylor** [comment]
> I just reached out to Clarivate to check on the status and got this response:
> 
> > Dear Chad,
> > Thank you for contacting Clarivate.
> > [snip]

Surely you can send a follow-up message asking for the reasons JOSS was rejected?

**Comment by chadagreene** [comment]
@MikeTaylor I agree, it would be nice to get more information, but I think any followup needs to come from the JOSS editors who submitted the application.

**Comment by danielskatz** [comment]
@arfon - can you say what email was received?

**Comment by arfon** [comment]
> Upon checking, I noticed that the journal has been recently rejected. Rejection emails has been sent out from the email of [alerts-noreply@clarivate.com](mailto:alerts-noreply@clarivate.com) and has be sent to who was listed as the Editorial Contact ([admin@theoj.org](mailto:admin@theoj.org)). The Editorial contact may also track the status of the journal on the Publisher Portal dashboard.

We received a desk rejection (based on an initial check) as we don't have :

- Editor titles and affiliations listed.
- A postal address for the publisher.

I'm planning on adding this information to the JOSS site in the coming weeks, but haven't got around to it yet. As it was a desk rejection, once we have made these changes we can immediately resubmit.

> @MikeTaylor I agree, it would be nice to get more information, but I think any followup needs to come from the JOSS editors who submitted the application.

Right. While I appreciate the enthusiasm here ❤ , I'd strongly prefer for the communications with Clarivate, Scopus, any external entity to come from the JOSS editorial team directly.

**Comment by MikeTaylor** [comment]
@arfon Oh, I strongly agree! When I wrote "Surely you can send a follow-up message asking for the reasons JOSS was rejected?" the "you" was meant to refer to the JOSS editorial team!

**Comment by sneakers-the-rat** [comment]
So if i've got the timeline right, we submitted last May and it took them this long to desk reject us for not having a listed address? (the editor titles and affiliations [do seem to be listed](https://joss.theoj.org/about)) And this is after a prior attempt at being indexed `n` (?) years ago? Seems like a slow walk to me. thanks to y'all for handling the work of submission and resubmission, even if we (speaking personally) don't love the proprietary index machine.

**Comment by JaGeo** [comment]
If there are any news, I would also be interested.

**Comment by fboehm** [comment]
What is the postal address for the publisher? I'm not sure what this term means, or what we might list for it. Are we expected to list Arfon's home mailing address? I'm not sure that that's a good idea...

**Comment by logological** [comment]
I've known other journals that were published by individuals (usually retired/former academics) and where the publisher's address reported in the journal was the publisher's home address.  So this situation isn't entirely unheard of.  Whether Arfon wants his home mailing address disclosed is something he'd have to decide for himself. (It's not clear to me whether WoS wants this address published in JOSS, or simply disclosed privately to Clarivate.)

If the address needs to be made public and Arfon doesn't want to give his home address, then some possible options would be:

* JOSS pays for a PO box at some location convenient for Arfon to occasionally check.
* Arfon or some trusted/long-term editor gets permission from their employer to use their office address, and remembers to occasionally check for incoming mail.

**Comment by rossmounce** [comment]
I'm pretty sure if one did some sleuthing one would find many instances of Clarivate indexed journals that give a PO box as their address. I don't have an example to hand, but I'd be willing to be there are many.

**Comment by mkhorton** [comment]
I believe there are "virtual PO box" options too, where physical mail can be scanned and forwarded electronically. For example see https://www.anytimemailbox.com which offers this for $5/mo (not an endorsement of this service, I have not used it).

**Comment by JaGeo** [comment]
I am just adding here again that many institutions check annual performances based on Web of Science-listed journals. Thus, this is a huge consideration when submitting a paper.

**Comment by danielskatz** [comment]
As the postal address for the publisher, perhaps we should be using NumFOCUS's address as our fiscal sponsor

**Comment by mkhorton** [comment]
This seems like a good suggestion @danielskatz, do you know who can "action" this?

**Comment by danielskatz** [comment]
We don't really have anyone specifically in charge of this, which means it kind of falls to @arfon at this point, though a volunteer could offer to take charge

**Comment by MartinBeseda** [comment]
@danielskatz Are there any news about the publisher's address? Now, I'm no member of the editorial board, but I'd be glad to help with some administration, if it's possible from my position :)

**Comment by danielskatz** [comment]
We think we should use the NumFOCUS address, as they are our fiscal sponsor. They are listed on https://numfocus.org as 
P.O. Box 90596
Austin, TX 78709

**Comment by MartinBeseda** [comment]
@danielskatz Great! If there's anything I could help with in this direction, don't hesitate to ask. And good luck anyway!

**Comment by danielskatz** [comment]
@MartinBeseda - to be honest, we need someone who wants to put some time into organizing this and making it happen...

**Comment by MartinBeseda** [comment]
@danielskatz Well, I should have some time for that, if it's acceptable for you.

**Comment by sneakers-the-rat** [comment]
The vibe I get on this is do-ocracy: if ya wanna do it, please do :)

**Comment by danielskatz** [comment]
I think that's quite right - go for it @MartinBeseda. I'm happy to comment on ideas and give feedback, or provide information...    Thanks!!

**Comment by MartinBeseda** [comment]
@sneakers-the-rat @danielskatz OK, let's do it! First of all, I'm afraid, I'll need some "official credentials", if I'm going to talk for the whole JOSS. Do you keep some position of "journal correspondent" or something like that? :D

Also, from what I understood, there's already a long communication with Clarivate in progress. Could I get a copy of some recent e-mail to be able to continue in-line? If not, I'm gonna just make a fresh "ticket".

Thank you for the info so far!

**Comment by jhelvy** [comment]
I just found this thread because I was considering a submission to JOSS but couldn't find it indexed anywhere, and Google brought me here. Good timing I guess as it seems this is happening like right now! I also came across #721 which is somewhat related, so if anything I'm adding this comment to make sure that issue is tagged here and also so that I get notified of replies as I'd like to keep up with the progress. The issue I referenced cites the declaration on https://sfdora.org/, (which I had never heard of before) which seems to advocate against the use of the Impact Factor metric for P&T decisions, etc. (I agree!). But even if JOSS is opposed to IFs, it still seems important that the journal be indexed by all the major orgs doing publication indexing. It makes it harder to find otherwise, and other metrics (e.g. citations) are still helpful, so I see being on these lists as more helpful than harmful. Not sure if there's anything I could do to help get it there, but happy to see others are working on it!

**Comment by JaGeo** [comment]
Just also to emphasize from my side: i have now two publications in JOSS that don't count as full publications at the institution where I work because of this missing entry in Web of Science. I can only submit further publications if JOSS is listed there.

**Comment by MartinBeseda** [comment]
@jhelvy @JaGeo Yes, being the Clarivate-registered "impacted" journal is important nowadays, literally up to the point, where some institutions don't consider the publications into employee's evaluations otherwise. So, we should better hurry :)

**Comment by arfon** [comment]
@MartinBeseda – thanks for offering to help out here. [This Google doc](https://docs.google.com/document/d/1m-DRFsFywRq9M61j1ho5fal5ca0hv1Jfq2faVNE8Mvs/edit?usp=sharing) has the most up to date commentary from me (and @diehlpk), currently framed around responses to the rejection letter from Scopus.

We submitted to Clarivate back in May 2023, but were rejected without a detailed response. 

> @sneakers-the-rat @danielskatz OK, let's do it! First of all, I'm afraid, I'll need some "official credentials", if I'm going to talk for the whole JOSS. Do you keep some position of "journal correspondent" or something like that? :D

@MartinBeseda – I would rather we didn't go down this route unless we can avoid it (I'd rather keep the communications from the admin@theoj.org email for example). 

My suggested next steps here would be to write up a solid response to the Scopus rejection and start with a resubmission there. Given we've been rejected by Web of Science in the last 2 years, I don't think we can reapply until May 2025 anyway.

**Comment by jhelvy** [comment]
Just saw #1352 ...this is bad ya'll. I'm 100% pro open-source, but the system isn't going to change if JOSS doesn't get on these lists. I haven't even participated yet in JOSS (as a reviewer or submitter), but the currency of promotion is publications, and if we can't "count" a JOSS pub then we aren't going to submit here, plain and simple. I'd love to see JOSS show the world that we can take our work back from the choke hold the publishers have on academia, but you have to play their game to get there. 

On a related note, I know some don't want a JOSS IF (e.g. #721), but without one only senior faculty with tenure will have the luxury of submitting to JOSS. Junior scholars who need numbers for promotion will not send their work here. Of course I'd love to see IFs disappear from T&P processes altogether, but that's not happening any time soon, and the organization as of now will not have any standing to push for those efforts if it's not even recognized. All this is to say, JOSS needs to get on all of these lists ASAP.

The [Google doc](https://docs.google.com/document/d/1m-DRFsFywRq9M61j1ho5fal5ca0hv1Jfq2faVNE8Mvs/edit?usp=sharing) of the response to the Web of Science rejection seems like this could be a quick turnaround. Basically, they rejected on some pretty bogus claims. JOSS has the data to show their claims are wrong. I'll leave some comments on the doc.

**Comment by MartinBeseda** [comment]
@arfon OK then, would you like the draft of the response in the same Google doc?

Also, I believe, that we should compare JOSS acceptance rates with some selected journal indexed under Scopus / WoS to see, what's their expected value. 

At worst, we could do a fixed number of "improvement attempts" after which the manuscript would be rejected, while encouraged to re-submit and continue from that point on, if that seems more "serious". Of course, this is a cumbersome approach I wouldn't personally prefer at all.

@jhelvy I completely agree.

Also, it seems, that problem with indexing under Google Scholar (mentioned in #130 ) reappeared unfortunately. I'll make a new issue for that one and link it here, as I believe, that while it's not connected to Wos / Clarivate, it's of equal importance nowadays.

**Comment by danielskatz** [comment]
Hi - I don't want to get into a dispute over the value of all these traditional metrics, but will just point out that 
1. JOSS is publishing about 1.4 papers per day, and this is increasing, 
2. all of the volunteer editors are at our limits, which in my opinion is limiting how much we can publish,
3. we will soon be seeking more editors to increase this

So while I see that JOSS could have more publications and have more impact on the field if we were indexed in some of these platforms, and I do want us to be indexed in them, I don't think the sky is falling or that not being indexed is actually harming JOSS today, since if we had more submissions, we couldn't deal with them.

Again in my opinion, we're also not trying to be something for all researchers; we focus on those researchers who want to use JOSS's process to make their software better, and to create a citable object so that they can get credit when their software is cited, and we seem to have found a large community of such researchers, from grad students to senior faculty to people in labs, government and industry.

**Comment by MartinBeseda** [comment]
@danielskatz First of all, I agree with most of your response here.

Where I differ is the notion of "not being indexed is not harming JOSS today", as I believe it does. It's one thing to have many papers / software to review and the other one to be considered reputable in general. What we don't want JOSS to become is an "easy venue", where most publications are accepted over time and people are trying to use it only for increasing their track record, while research institutions are aware of that and consider it a low-value journal close to e.g. predatory ones. So, I believe, we should aim to be more involved with the "traditional metrics" as JOSS is a scientific journal, even if with its specific characteristics (which is more or less applicable to every journal out there). In other words, I do think, that if we aim to stay relevant in the long term, we should try to "beat the game", or, at least, be involved in it. And eventually, this is also directly connected to "getting credit" you mentioned, as the credit should be pretty much universal, not rejected in some institutions as "bad journal publication".

P.s.: If you're seeking for more editors, I'd join the group, if considered somewhat helpful :)

**Comment by jedbrown** [comment]
Note that Elsevier and Springer Nature (along with many professional organizations and national councils) are signatories of the [Declaration on Research Assessment (DORA)](https://sfdora.org/read/), which vows to avoid metrics like impact factor and focus on scientific content. My department incorporated some of this language in revised promotion and tenure guidelines, and were able to hire on that basis. I understand the process takes time, but change is achievable.

<details>

> ## For institutions
>
> 4. Be explicit about the criteria used to reach hiring, tenure, and promotion decisions, clearly highlighting, especially for early-stage investigators, that the scientific content of a paper is much more important than publication metrics or the identity of the journal in which it was published.
>
> 5. For the purposes of research assessment, consider the value and impact of all research outputs (including datasets and software) in addition to research publications, and consider a broad range of impact measures including qualitative indicators of research impact, such as influence on policy and practice.
>
> ## For publishers
>
> 6. Greatly reduce emphasis on the journal impact factor as a promotional tool, ideally by ceasing to promote the impact factor or by presenting the metric in the context of a variety of journal-based metrics (e.g., 5-year impact factor, EigenFactor [8], SCImago [9], h-index, editorial and publication times, etc.) that provide a richer view of journal performance.
>
> 7. Make available a range of article-level metrics to encourage a shift toward assessment based on the scientific content of an article rather than publication metrics of the journal in which it was published.
>
> 8. Encourage responsible authorship practices and the provision of information about the specific contributions of each author.
>
> 9. Whether a journal is open-access or subscription-based, remove all reuse limitations on reference lists in research articles and make them available under the Creative Commons Public Domain Dedication [10].

</details>

**Comment by danielskatz** [comment]
@MartinBeseda - the JOSS call for editors will be posted in https://blog.joss.theoj.org and shared via social media - I encourage you to apply.

**Comment by danielskatz** [comment]
On the other hand, I have to disagree with part of your comment
> What we don't want JOSS to become is an "easy venue", where most publications are accepted over time and people are trying to use it only for increasing their track record

We don't want JOSS to be an "easy venue" that publishes low-quality work, but we do want JOSS to publish high quality work, no matter how much of it is submitted. Today, we reject about 20-25% of submissions as being out-of-scope, and of the submissions that enter review, about 95% are published. We do not consider a high rejection rate a plus, and want to publish all high-quality work.

If all our submissions were in scope, and either high-quality at the start or improved to that point during the review process, and we published all of them, I would consider this a success, though "traditional metrics" would see this as bad.

This is a fundamental difference between JOSS and traditional journals, and a way we are more like open source software than traditional publishing, even though we have elements of both.  (As an example of this, imagine if open source software projects were valued based on the rate of PRs and issues they rejected, rather than how useful their software was.)

**Comment by jhelvy** [comment]
The reviewer comments about acceptance rates being high is a rather ridiculous reason to reject JOSS. If the JOSS acceptance rate is ~75%, that's higher than the mean & median, but certainly not the highest:

![Screen Shot 2024-10-22 at 14 01 51](https://github.com/user-attachments/assets/2117ebae-016a-4bbd-a799-0ec24590b5d3)

I screened that from [this paper](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3526365), which was referenced in [this article](https://scientific-publishing.webshop.elsevier.com/publication-process/journal-acceptance-rates/). 

And acceptance rate is only one of the metrics, which should naturally be high for a journal like JOSS. The majority of the effort is in building the software, which I would assume has gotten relatively mature / stable by the time that someone would want to make a submission to JOSS. So it's much more likely that the acceptance rate would be on the high end as the work is coming in at a more mature state (correct me if I'm wrong).

**Comment by MartinBeseda** [comment]
@danielskatz Fair enough, the approach "accept, when good enough" is something I completely agree with.

@jhelvy Great find! I believe, that specifically this kind of argument should be posed for our next attempt.

**Comment by danielskatz** [comment]
FYI (from @tarleb on JOSS slack): https://elifesciences.org/inside-elife/16afe6ec/update-on-elife-s-indexing-status-at-web-of-science

**Comment by arfon** [comment]
> @arfon OK then, would you like the draft of the response in the same Google doc?

Apologies for the very slow response. Yes please @MartinBeseda !

**Comment by MartinBeseda** [comment]
@arfon Apologies also on my side, I was ill... I'll have a look at the response during this week.

**Comment by jmestret** [comment]
Hi, is there any update on this? Thanks!

**Comment by r03ert0** [comment]
hello! any news on this?

**Comment by julian-belina** [comment]
Is there any update on this? I found this Elsevier journal called "SoftwareX" which is listed by scopus. I don't understand why JOSS is rejected when it has similar metrics https://www.scopus.com/sourceid/21100422153

**Comment by JaGeo** [comment]
Yes. For now, I am therefore have moved software publications elsewhere...

**Comment by aloctavodia** [comment]
Let me rephrase my question. Is there something I/we can do to help?

**Comment by arfon** [comment]
Hi folks, thanks for the nudge on this. We resubmitted today. 🤞 things go better this time.

**Comment by PEEKPerformer** [comment]
@arfon Any updates? Hopefully everything went well.

**Comment by arfon** [comment]
No news yet sorry.

**Comment by sarangbhagwat** [comment]
Hi @arfon, any updates on the status of your re-submittal? Thanks for all the effort you're putting into making this happen. (Apologies if this has already been resolved elsewhere that I haven't seen.)
